### DISCUSSION TITLE
Node red code for oee calculation

### DISCUSSION THREAD

**Discussion by MasterKensh** [community/community#132976] [Programming Help]
### Select Topic Area

Question

### Body

hello, I am designing an injection machine oee monitor using node red form my 3year project for campus but the values for performance, availability and quality are not displaying. I need help

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @MasterKensh ! 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
